node/popn.go

```diff
@@ -124,6 +127,11 @@ type Options struct {
 	CancelFunc context.CancelFunc
 }
 
+type MetricsRecorder interface {
+    Record(string, map[string]string, map[string]interface{})
+		URL() string
```

Comment:
What's that for?

Reply:
just a handy thing for logging what database URL we're hitting up